sgkit/testing.py

```diff
@@ -0,0 +1,56 @@
+from typing import Optional
+
+import numpy as np
+
+from .api import create_genotype_call_dataset
+
+
+def simulate_genotype_call_dataset(
+    n_variant: int,
+    n_sample: int,
+    n_ploidy: int = 2,
+    n_allele: int = 2,
+    n_contig: int = 1,
+    seed: Optional[int] = None,
+):
+    """Simulate genotype calls and variant/sample data
```

Comment:
This is a good idea, and I agree it's a useful thing to expose to let us define examples and so on. I'd just add a note emphasising that the data isn't in any way realistic, and it shouldn't be used for anything other than testing and examples.

Reply:
Will do! good point.

Reply:
Changed in https://github.com/pystatgen/sgkit/pull/36/commits/5815cac9ae12c3211720acf475573ee78061a9c1